lib/AL_USDMaya/AL/usdmaya/nodes/wrapProxyShape.cpp

```diff
@@ -0,0 +1,362 @@
+//
+// Copyright 2017 Animal Logic
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.//
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "AL/usdmaya/nodes/ProxyShape.h"
+
+#include <boost/python/args.hpp>
+#include <boost/python/def.hpp>
+#include <boost/python.hpp>
+
+#include "AL/usdmaya/Utils.h"
+
+#include "maya/MBoundingBox.h"
+#include "maya/MFnDagNode.h"
+#include "maya/MFnDependencyNode.h"
+#include "maya/MDagModifier.h"
+#include "maya/MDGModifier.h"
+#include "maya/MSelectionList.h"
+
+#include "pxr/base/tf/pyEnum.h"
+#include "pxr/base/tf/pyResultConversions.h"
+
+#include <memory>
+
+//using namespace std;
+//using namespace boost::python;
+//using namespace boost;
+
+
+using AL::usdmaya::nodes::ProxyShape;
+using boost::python::reference_existing_object;
+using boost::python::object;
+
+namespace {
+  struct MBoundingBoxConverter
+  {
+    // to-python conversion of const PxrUsdMayaXformOpClassification.
```

Comment:
PxrUsdMayaXformOpClassification->MBoundingBoxConverter

Reply:
Not sure what you mean by this note - are you saying you want to move the bounding-box-conversion code to PxrUsdMayaXformOpClassification, on the USD side of things?

Reply:
no - i thought it was an erroneous comment ie. you meant to say "// to-python conversion of const MBoundingBoxConverter" - but maybe I was wrong?